index.html

```diff
@@ -66,6 +66,285 @@ <h1 class="primary-text">I’m Abraham <br>Glad to see <br>you!</h1>
+            </li>
+          </ul>
+        </div>
+        <div class="project-text">
+          <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+        </div>
+
+        <!-- programming language used -->
+        <ul class="project-program-laguages">
+          <li class="program-language">
+            <p>html</p>
+          </li>
+          <li class="program-language">
+            <p>css</p>
+          </li>
+          <li class="program-language-javascript">
+            <p>javascript</p>
+          </li>
+        </ul>
+        <!-- project call to action section -->
+        <button type="submit" class="call-to-action">See project</button>
+
+      </section>
+      <!-- project four end -->
+
+      <div class="workDivider"></div>
+      <!-- about me section starts hereby -->
+      <section class="aboutMeContainer">
+        <div class="intro-aboutMe">
+          <div class="text-container">
+            <h2 class="primary-text-about">About<br>Myself</h2>
+            <p class="secondary-text">I’m a software developer! I can help <br> you build a product , feature or website
+              <br>
+              Look through some of my work and <br>experience! If you like what you see and have <br>aproject you need
+              coded,
+              don’t hestiate<br>to contact me.
+            </p>
+          </div>
+
+          <!-- call to action text -->
+
+          <!-- <h2 class="link-text link-container">LET’S CONNECT</h2> -->
+          <a href="#" class="link-text">LET’S CONNECT</a>
+
+
+          <!-- social media icons -->
+          <div>
+            <ul class="social-container">
+              <li>
+                <a href="https://twitter.com/_Cabraham" target="_blank" rel="noopener noreferrer"><img
+                    class="social twitter" src="./assets/icons/twitter.svg" alt="twitter"></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/abrahamchristopher/" target="_blank" rel="noopener noreferrer"><img
+                    class="social linkedin move" src="./assets/icons/Linkedinicon.svg" alt="linkedin"></a>
+              </li>
+              <li>
+                <a href="" target="_blank" rel="noopener noreferrer">
+                  <img class="social meduim move" src="./assets/icons/medium.svg" alt="medium"></a>
+              </li>
+              <li>
+                <a href="https://github.com/Cabraham1" target="_blank" rel="noopener noreferrer">
+                  <img class="social github move" src="./assets/icons/github.svg" alt="github"></a>
+              </li>
+              <li>
+                <a href="" target="_blank" rel="noopener noreferrer">
+                  <img class="social agenllist move" src="./assets/icons/angellist.svg" alt="agenllist"></a>
+              </li>
+            </ul>
+          </div>
+          <button type="submit" class="call-to-action">Get my resume</button>
```

Comment:
- [ ] [REQUIRED] So close 👍🏼  Please make the `text` layout design like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=848ucIcBgTd3IXHT-0) 😃 

![image](https://user-images.githubusercontent.com/63915024/212111073-73784d09-e20a-4676-8ce5-abd00981d1a1.png)

Reply:
done